lib/ansible/modules/cron.py

```diff
@@ -73,28 +73,28 @@
     default: no
   minute:
     description:
-      - Minute when the job should run ( 0-59, *, */2, etc )
+      - Minute when the job should run ( 0-59, *, */2, etc ).
```

Comment:
```suggestion
      - Minute when the job should run (C(0-59), C(*), C(*/2), etc.).
```
Same for the others.

Reply:
good spot! fixed